<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogicalBalls</class>
 <widget class="QMainWindow" name="LogicalBalls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>93</x>
      <y>170</y>
      <width>614</width>
      <height>314</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="selectColorText">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Выберите цвет:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QComboBox" name="colorSelecting_3">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
       <item>
        <property name="text">
         <string>Синий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Красный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Жёлтый</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QFrame" name="frame_1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="colorSelecting_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
       <item>
        <property name="text">
         <string>Синий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Красный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Жёлтый</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="selectColorText_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Выберите цвет:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QFrame" name="frame_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="selectColorText_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Выберите цвет:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QComboBox" name="colorSelecting">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::NoFocus</enum>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="statusTip">
        <string/>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
       <property name="accessibleName">
        <string/>
       </property>
       <property name="accessibleDescription">
        <string/>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentText">
        <string/>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
       <property name="modelColumn">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>Синий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Красный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Жёлтый</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QFrame" name="frame_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="checkButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="text">
        <string>Проверить</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="ruleGame">
    <property name="geometry">
     <rect>
      <x>93</x>
      <y>56</y>
      <width>611</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>490</y>
      <width>200</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Запуск</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Меню</string>
    </property>
    <widget class="QMenu" name="submenu">
     <property name="title">
      <string>Выбор сложности</string>
     </property>
     <addaction name="changeHard"/>
    </widget>
    <addaction name="submenu"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="changeHard">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Усложнённый</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
